vendor/bat-native-ledger/include/bat/ledger/publisher_info.h

```diff
@@ -52,6 +52,11 @@ LEDGER_EXPORT enum PUBLISHER_EXCLUDE_FILTER {
   FILTER_ALL_EXCEPT_EXCLUDED = 3
 };
 
+LEDGER_EXPORT enum PUBLISHER_PERCENT_FILTER {
```

Comment:
why do we need an enum for this?

Reply:
seems like we should just default to 100% and the filter should `<=` in the DB

Reply:
we use enums for both excluded and month in the filter so it seemed to make sense to use one here.